src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1958,36 +1958,53 @@ namespace winrt::TerminalApp::implementation
         _HookupKeyBindings(_settings.ActionMap());
 
         // Refresh UI elements
-        auto profiles = _settings.ActiveProfiles();
-        for (const auto& profile : profiles)
+
+        // Mapping by GUID isn't _excellent_ because the defaults profile doesn't have a stable GUID; however,
+        // when we stabilize its guid this will become fully safe.
+        std::unordered_map<winrt::guid, std::pair<Profile, TerminalSettingsCreateResult>> _profileGuidSettingsMap;
+
+        // Include the Defaults profile for consideration
+        const auto profileDefaults{ _settings.ProfileDefaults() };
+        _profileGuidSettingsMap.insert_or_assign(profileDefaults.Guid(), std::pair<Profile, TerminalSettingsCreateResult>{ profileDefaults, nullptr });
```

Comment:
Perhaps silly, but does the type inference let you get away with `std::pair { profileDefaults, nullptr }`?

Reply:
It does! I had tried simply `{ ... }` but wasn't aware that I could leave off the template args to be deduced by the context they appear in (rather than the contents of the braces)!